src/components/Title.tsx

```diff
@@ -0,0 +1,11 @@
+interface Props {
+	title: string;
+}
+
+export const Title = ({ title }: Props) => {
+	return (
+		<div>
+			<h2 className="text-balck text-5xl text-center mt-8 mb-6 ">{title}</h2>
```

Comment:
Jen mi přijde zvláštní, že se to jmenuje Title takže bych očekával H1 a ne H2, ale neříkám, že to není dobře a je to maličkost... to asi nech